sample/src/main/java/io/ona/kujaku/sample/activities/FociBoundaryActivity.java

```diff
@@ -43,14 +44,29 @@ protected void onCreate(Bundle savedInstanceState) {
                     .setBoundaryColor(Color.RED)
                     .setBoundaryWidth(6f);
 
-            kujakuMapView.addLayer(builder.build());
+            BoundaryLayer boundaryLayer = builder.build();
+            kujakuMapView.addLayer(boundaryLayer);
 
             kujakuMapView.getMapAsync(new OnMapReadyCallback() {
                 @Override
                 public void onMapReady(MapboxMap mapboxMap) {
                     // Zoom to the position
                     mapboxMap.setStyle(Style.MAPBOX_STREETS);
                     mapboxMap.easeCamera(CameraUpdateFactory.newLatLngZoom(new LatLng(-14.105596638939968, 32.60676728350983), 15d));
+
+                    new Thread(() -> {
+                        try {
+                            Thread.sleep(6000);
```

Comment:
Why not display a button at the bottom for changing to another focus area. So that user is aware what is happening.

If you decide with approach then animate to the new focus area .

Reply:
Cool, that's a good idea